services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptTrials/llmPromptTrials.tsx

```diff
@@ -0,0 +1,216 @@
+  //     return <Link to={props}>Add Plagiarism Rule</Link>;
+  //   }
+  //   if(rulePrompts[conjunction]) {
+  //     const { id } = rulePrompts[conjunction];
+  //     props.state.promptIds = [id];
+  //     return <Link to={props}>{`Add ${titleCase(conjunction)} Plagiarism Rule`}</Link>;
+  //   }
+  //   return <Link to={props}>Add Plagiarism Rule</Link>;
+  // }
+  //
+  // if(!plagiarismRulesData) {
+  //   return(
+  //     <div className="loading-spinner-container">
+  //       <Spinner />
+  //     </div>
+  //   );
+  // }
+  //
+  // if(plagiarismRulesData.error) {
+  //   return(
+  //     <div className="error-container">
+  //       <Error error={`${plagiarismRulesData.error}`} />
+  //     </div>
+  //   );
+  // }
+  //
+  // // Header labels are redundant so passing empty strings and hiding header display
+  // const dataTableFields = [
+  //   { name: "", attribute:"field", width: "200px" },
+  //   { name: "", attribute:"value", width: "800px" }
+  // ];
+  // const becauseRuleLink = getAddRuleLink(BECAUSE);
+  // const butRuleLink = getAddRuleLink(BUT);
+  // const soRuleLink = getAddRuleLink(SO);
+  // const becausePlagiarismRule = getFormattedRows(plagiarismRulesData, BECAUSE);
+  // const butPlagiarismRule = getFormattedRows(plagiarismRulesData, BUT);
+  // const soPlagiarismRule = getFormattedRows(plagiarismRulesData, SO);
+  // const becauseDisabled = getButtonClass(BECAUSE);
+  // const butDisabled = getButtonClass(BUT);
+  // const soDisabled = getButtonClass(SO);
+  // const oneRuleWarning = 'There can only be one plagiarism rule per conjunction. Please click "View" below if you would like to update the plagiarized text. Red text means that the plagiarism text does not match the associated passage.'
+  //
+  // return(
+  //   <div className="rules-container">
+  //     {renderHeader(activityData, 'Plagiarism Rules')}
+  //     <section className="plagiarism-section" id="first-plagiarism-section">
+  //       <button className={`quill-button-archived fun primary contained add-rule-button ${becauseDisabled}`} type="submit">{becauseRuleLink}</button>
+  //       {becauseDisabled && <p className="one-rule-warning">{oneRuleWarning}</p>}
+  //       {renderTable(becausePlagiarismRule)}
+  //     </section>
+  //     <section className="plagiarism-section">
+  //       <button className={`quill-button-archived fun primary contained add-rule-button ${butDisabled}`} type="submit">{butRuleLink}</button>
+  //       {butDisabled && <p className="one-rule-warning">{oneRuleWarning}</p>}
+  //       {renderTable(butPlagiarismRule)}
+  //     </section>
+  //     <section className="plagiarism-section">
+  //       <button className={`quill-button-archived fun primary contained add-rule-button ${soDisabled}`} type="submit">{soRuleLink}</button>
+  //       {soDisabled && <p className="one-rule-warning">{oneRuleWarning}</p>}
+  //       {renderTable(soPlagiarismRule)}
+  //     </section>
+  //   </div>
+  // );
```

Comment:
is all of this supposed to be commented out?

Reply:
Whoops, this shouldn't have been in this PR at all. Removed!